acctest/stubserver.go

```diff
@@ -0,0 +1,39 @@
+package acctest
```

Comment:
This is the only `.go` file without public domain notice at the top of it. We either need to have it in all files or only in the license file.